File: Solved_001_025/Problem-002/sol.cs
using System;

public class Test {
	public static void Main() {
		int limit = 4000000;
		int previous = 1;
		int result = 0;
		int x = 0;
		
		for (int i = 1; i <= limit; i += previous) {
			if(x > 0) {
				previous = i - previous;
			}
			
			x++;
			
			if (i % 2 == 0) {
				result += i;
			}
		}
		
		Console.WriteLine(result);
	}
}

File: Solved_001_025/Problem-001/sol.c
#include <stdio.h>

int main(void) {
	int result = 0;

	for (int i = 1; i < 1000; i++) {
		if (i % 3 == 0 || i % 5 == 0) {
			result += i;
		}
	}

	printf("%d\n", result);

	return 0;
}
